src/test/integration/IntegrationDeployer.t.sol

```diff
@@ -23,24 +24,22 @@ import "src/test/mocks/ETHDepositMock.sol";
 import "src/test/integration/mocks/BeaconChainOracleMock.t.sol";
 import "src/test/integration/mocks/BeaconChainMock.t.sol";
 
-import "src/test/integration/User.t.sol";
+import "src/test/integration/users/User.t.sol";
+import "src/test/integration/users/User_M1.t.sol";
 
-abstract contract IntegrationDeployer is Test, IUserDeployer {
+import "script/ExistingDeploymentParser.sol";
+
+abstract contract IntegrationDeployer is ExistingDeploymentParser {
 
     Vm cheats = Vm(HEVM_ADDRESS);
 
-    // Core contracts to deploy
-    DelegationManager public delegationManager;
-    StrategyManager public strategyManager;
-    EigenPodManager public eigenPodManager;
-    PauserRegistry pauserRegistry;
-    Slasher slasher;
-    IBeacon eigenPodBeacon;
-    EigenPod pod;
-    DelayedWithdrawalRouter delayedWithdrawalRouter;
+    // Fork ids for specific fork tests
+    bool isUpgraded;
+    uint256 mainnetForkBlock = 19_280_000;
+    uint256 mainnetForkId;
+    uint256 goerliForkBlock = 10_575_000;
+    uint256 goerliForkId;
```

Comment:
Let's get rid of the goerli references, since that's unstable (and also slightly out of date with the dev branch)